src/MoBi.UI/Views/EditParameterListView.cs

```diff
@@ -120,7 +120,7 @@ public override void InitializeBinding()
 
          _pathBinder.InitializeBinding(_gridViewBinder);
          _gridViewBinder.Bind(dto => dto.Value)
-            .WithFormat(dto => dto.ParameterFormatter())
+            .WithFormat(dto => dto.ParameterFormatter(checkForEditable:false))
```

Comment:
Thsi is the other issue that was showing different value for NaN depending in editable or not. MoBi does not care for this so I want to make sure it's consistent.